packages/flutter/test/material/slider_test.dart

```diff
@@ -3437,6 +3437,62 @@ void main() {
     );
   }, variant: TargetPlatformVariant.desktop());
 
+  testWidgets('Event on Slider should perform no-op if already unmounted',
+      (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Event on Slider should perform no-op if already unmounted', (WidgetTester tester) async {
```

This is fine to be in one line.